features/facebookImport/src/views/overview/overview.css

```diff
@@ -73,7 +73,7 @@ h4 {
 
 .overview .footer {
     width: 100%;
-    position: absolute;
+    position: fixed;
```

Comment:
Is that related?

Reply:
Sorry, it's not. This commit accidentally sneaked into this branch when I created it.